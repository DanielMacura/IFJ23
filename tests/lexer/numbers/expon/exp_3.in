let x = 1.23e-12
x = 1.444e+123
x = 111