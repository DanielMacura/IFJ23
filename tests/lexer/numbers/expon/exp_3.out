KW_LET
VARIABLE
EQUALS
FLOAT:0.000000000001230000000
EOL
VARIABLE
EQUALS
INTEGER:-2147483648
EOL
VARIABLE
EQUALS
INTEGER:111
EOF
